Makefile

```diff
@@ -99,3 +99,35 @@ examples/openshift/manifests: examples/openshift/main.jsonnet jsonnet/controller
 	jsonnetfmt -i examples/openshift/main.jsonnet
 	jsonnet -m examples/openshift/manifests examples/openshift/main.jsonnet | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml' -- {}
 	find examples/openshift/manifests -type f ! -name '*.yaml' -delete
+
+KIND_CONFIG_FILE := testdata/kind-cluster.yaml
+
+kind:
+ifeq (, $(shell which kind))
+	go install sigs.k8s.io/kind@v0.23.0
+KIND=$(GOBIN)/kind
+else
+CONTROLLER_GEN=$(shell which kind)
+endif
+
+helm-repo-add:
+	helm repo add grafana https://grafana.github.io/helm-charts || true
+
+helm-repo-update: helm-repo-add
+	helm repo update grafana
+
+ensure-kind-cluster: kind
+	@if ! kind get clusters | grep -q "^mimir$$"; then \
+		kind create cluster --config $(KIND_CONFIG_FILE); \
+	else \
+		echo "Cluster is already configured"; \
+	fi
```

Comment:
While this is nice if people use kind there are many different ways of running Kubernetes these days. minikube, k3s, or simply via Docker on Mac. Let's just assume there's going to be _a_ Kubernetes cluster available. 
It also makes sense to avoid including it in the `Makefile` and instead include these scripts in a Mimir helper script. 

I'm thinking similar to this: https://github.com/pyrra-dev/pyrra/tree/main/examples/docker-compose

Reply:
Makes sense. I used kind because I could assign a fixed `NodePort` to easily route the traffic to the service, but I can also just add a port-forward command.

I was also torn between adding it to the Makefile vs a script, will do that!

Reply:
I added `examples/mimir/` which includes a README with the instructions. Let me know what you think.